wgpu-types/src/lib.rs

```diff
@@ -292,8 +323,7 @@ bitflags::bitflags! {
     ///
     /// `ShaderStage::VERTEX | ShaderStage::FRAGMENT`
     #[repr(transparent)]
-    #[cfg_attr(feature = "trace", derive(Serialize))]
-    #[cfg_attr(feature = "replay", derive(Deserialize))]
+    #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
```

Comment:
why are we changing this one?

Reply:
This is now used in a couple places (I think RenderCommand) that requires both if it needs either.